server/sonar-server/src/main/java/org/sonar/server/project/ws/BulkDeleteAction.java

```diff
@@ -72,7 +71,7 @@ public void define(WebService.NewController context) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
-    userSession.checkPermission(UserRole.ADMIN);
+    userSession.checkLoggedIn().checkIsRoot();
```

Comment:
description should be fixed, root is now required, not system administration... or is that the name we give to root?